src/client/css/_gallery-list.scss

```diff
@@ -1,108 +1,96 @@
 .gallery-pagination {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
-
+  
   margin: 0 auto;
   max-width: 350px;
 
   padding: 25px 25px 0;
 
+  a:first-child {
+    flex-grow: 1;
+  }
+  span {
+    flex-grow: 1;
+    text-align: center;
+  }
+
+  a:last-child {
+    flex-grow: 1;
+    text-align: right;
+  }
+
+
   a:hover {
     cursor: pointer;
     text-decoration: underline;
   }
 }
-
 .gallery-list {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  max-width: 1300px;
+  display: grid;
+  grid-template-columns: repeat(4,300px);
+  justify-content: center;
+  grid-gap: 5px;
+  @media screen and (max-width: 1245px) {
+    grid-template-columns: repeat(auto-fill, 300px);
+  }
 
   .gallery-card {
+    display: inline-block;
     margin: 5px;
-
     box-shadow: 0 2px 4px 0 #aaa;
     border-radius: 5px;
     transition: 0.3s;
+    width: 300px;
+    height: 200px;
 
     @media (max-width: 450px) {
-      width: 95vw;
+      // width: 95vw;
     }
 
     a {
       margin: 0;
       text-decoration: none;
-      overflow: hidden;
-
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      justify-content: center;
 
       img {
         margin: 0;
         padding: 0;
         border-radius: 5px;
         object-fit: cover;
 
-        /* using fixed height due to the variety of dimensions .. */
         height: 200px;
         width: 300px;
 
         @media (max-width: 450px) {
-          width: 95vw;
+          //width: 95vw;
         }
       }
-
       .title {
-        display: flex;
-        flex-direction: row;
-        align-items: center;
-
-        width: 100%;
+        position: relative;
```

Comment:
Are you sure about using `relative` rather than flexbox to perform the positioning?